samples/document-security/README.md

```diff
@@ -0,0 +1,270 @@
+---
+name: ChatGPT + Enterprise data + document security
+description: This guide demonstrates how to add an optional login and document level access control system to the sample. This system can be used to restrict access to indexed data to specific users.
+languages:
+- python
+- typescript
+- bicep
+- azdeveloper
+products:
+- azure-openai
+- azure-cognitive-search
+- azure-app-service
+- azure
+page_type: sample
+urlFragment: azure-search-openai-demo-document-security
+---
+# Setting up optional login and document level access control
+
```

Comment:
Possible first paragraph for this one:

"The [azure-search-openai-demo](https://github.com/Azure-Samples/azure-search-openai-demo/) project can set up a full RAG chat app on Azure AI Search and OpenAI so that you can chat on custom data, like internal enterprise data or domain-specific knowledge sets. For full instructions on setting up the project, consult the [main README](/README.md), and then return here for detailed instructions on configuring login and access control."